nifi-assembly/pom.xml

```diff
@@ -878,6 +878,18 @@ language governing permissions and limitations under the License. -->
                     <version>1.11.0-SNAPSHOT</version>
                     <type>nar</type>
                 </dependency>
+                <dependency>
+                    <groupId>org.apache.nifi</groupId>
+                    <artifactId>nifi-easyrules-nar</artifactId>
```

Comment:
Do we need this NAR here? I would think we'd interact with the Rules Engine using the `RulesEngineService` interface in `nifi-rules-engine-service-api`. Or is it included so there's a guaranteed implementation of the interface? In that case I'm good with it, or perhaps we should consider trivial implementations in the API, such as `AlwaysFireRulesService` and/or `NeverFireRulesService`.

Reply:
I included it to always have some implementation available